index.js

```diff
@@ -10,7 +10,9 @@ function StylusCompiler(cfg) {
   this.rootPath = cfg.paths.root;
   this.config = (cfg.plugins && cfg.plugins.stylus) || {};
   this.getDependencies = progeny({
-    rootPath: this.rootPath
+    rootPath: this.rootPath,
+    extension: this.extension,
```

Comment:
is specifying `extension` necessary? won't progeny auto-detect it from the filepath?

I suppose it won't do any harm though